x/feegrant/keeper/keeper.go

```diff
@@ -24,16 +25,37 @@ type Keeper struct {
 	cdc          codec.BinaryCodec
 	storeService store.KVStoreService
 	authKeeper   feegrant.AccountKeeper
+	Schema       collections.Schema
+	// FeeAllowance key: grantee+granter | value: Grant
+	FeeAllowance collections.Map[collections.Pair[sdk.AccAddress, sdk.AccAddress], feegrant.Grant]
+	// FeeAllowanceQueue key: expiration time+grantee+granter | value: bool
+	FeeAllowanceQueue collections.Map[collections.Triple[time.Time, sdk.AccAddress, sdk.AccAddress], bool]
 }
 
 var _ ante.FeegrantKeeper = &Keeper{}
 
 // NewKeeper creates a feegrant Keeper
 func NewKeeper(cdc codec.BinaryCodec, storeService store.KVStoreService, ak feegrant.AccountKeeper) Keeper {
+	sb := collections.NewSchemaBuilder(storeService)
+
 	return Keeper{
 		cdc:          cdc,
 		storeService: storeService,
 		authKeeper:   ak,
+		FeeAllowance: collections.NewMap(
+			sb,
+			feegrant.FeeAllowanceKeyPrefix,
+			"allowances",
+			collections.PairKeyCodec(sdk.LengthPrefixedAddressKey(sdk.AccAddressKey), sdk.LengthPrefixedAddressKey(sdk.AccAddressKey)), //nolint: staticcheck // sdk.LengthPrefixedAddressKey is needed to retain state compatibility
+			codec.CollValue[feegrant.Grant](cdc),
+		),
+		FeeAllowanceQueue: collections.NewMap(
+			sb,
+			feegrant.FeeAllowanceQueueKeyPrefix,
+			"allowances_queue",
+			collections.TripleKeyCodec(sdk.TimeKey, sdk.LengthPrefixedAddressKey(sdk.AccAddressKey), sdk.LengthPrefixedAddressKey(sdk.AccAddressKey)), //nolint: staticcheck // sdk.LengthPrefixedAddressKey is needed to retain state compatibility
+			collections.BoolValue,
```

Comment:
are we able to only backport the changes from your pr on main? There isnt a reason to backport this